cfgov/unprocessed/apps/paying-for-college/js/college-costs/util/url-parameter-utils.js

```diff
@@ -110,15 +110,16 @@ const models = {
  * @returns {object} key-value pairs of the queryString
  */
 function getQueryVariables() {
-  const query = window.location.search.substring(1);
-  const pairs = query.split('&');
-  const queryVariables = {};
-  pairs.forEach((elem) => {
-    const pair = elem.split('=');
-    const key = decodeURIComponent(pair[0]);
-    const value = decodeURIComponent(pair[1]);
-    queryVariables[key] = value;
-  });
+  let queryVariables = {};
```

Comment:
Could this be `const queryVariables = {}`? It looks like you're setting the values in the object later on, but aren't setting the object itself. `yarn lint` might catch this too.